src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -128,14 +135,45 @@ def init_from_file(self) -> None:
             profile_jsonc = commentjson.load(fileobj)
 
         self.profiles = profile_jsonc.get("profiles", {})
-        self.defaults = profile_jsonc.get("defaults", {})
         self.schema_property = profile_jsonc.get("$schema", None)
+        self.defaults = profile_jsonc.get("defaults", {})
+        self.jsonc = profile_jsonc
 
+        if self.schema_property and validate_schema:
+            self.validate_schema(config_type, validate_schema)
         # loading secure props is done in load_profile_properties
         # since we want to try loading secure properties only when
         # we know that the profile has saved properties
         # self.load_secure_props()
 
+    def validate_schema(
+        self,
+        config_type: str,
```

Comment:
Could you remove the `config_type` parameter since it's no longer used in this function?